src/Upload.php

```diff
@@ -86,13 +86,14 @@ public function upload(string $pid, ?string $tableId = null): void
 
         try {
             $retryProxy->call([$webDav, 'uploadZip'], [$this->files, $webDav->getUrl() . $folderName]);
```

Comment:
I would consider adding logger also to retry proxy. As there is large fluctuation in duration, it could be because of upload retries.

Reply:
All right, and it's even logging `info` when it's retrying :+1: